package.json

```diff
@@ -5,16 +5,24 @@
   "description": "actual syncing server",
   "type": "module",
   "scripts": {
-    "start": "node app",
+    "start": "tsx app",
+    "docker:artifacts": "chmod +x docker/download-artifacts.sh && ./docker/download-artifacts.sh",
+    "docker:build-common": "yarn docker:artifacts && docker build --build-arg GITHUB_TOKEN=$(gh auth token) -t actual-server-dev",
+    "docker:build-edge": "yarn docker:build-common -f docker/edge-ubuntu.Dockerfile .",
+    "docker:build-edge-alpine": "yarn docker:build-common -f docker/edge-alpine.Dockerfile .",
+    "docker:build-stable": "yarn docker:build-common -f docker/stable-ubuntu.Dockerfile .",
+    "docker:build-stable-alpine": "yarn docker:build-common -f docker/stable-alpine.Dockerfile .",
+    "docker:run": "docker run --rm -p 5006:5006 actual-server-dev",
     "lint": "eslint . --max-warnings 0",
```

Comment:
I built all the docker images and ran them, making sure the server ran and was accessible with `curl`.